luminex/webapp/luminex/LeveyJenningsPlotHelpers.js

```diff
@@ -121,16 +121,31 @@ LABKEY.LeveyJenningsPlotHelper.renderPlot = function(config)
 {
     var plotData = [];
     var records = config.store.getRange();
+    var otherHoverProps = ['Network', 'AssayType', 'ExpPerformer', 'AcquisitionDate'];
 
     var _pushData = function(record)
     {
-        plotData.push({
+        var data = {
             xLabel: record.get('NotebookNo'),
             pointColor: record.get('LotNumber'),
             value: record.get(config.plotType),
             gsMean: record.get('GuideSet' + config.plotType + 'Average'),
             gsStdDev: record.get('GuideSet' + config.plotType + 'StdDev')
+        };
+
+        // add some other values to the data object for the hover display
+        Ext.each(otherHoverProps, function(prop) {
+            var val = record.get(prop);
+
+            // convert values that are date objects to a display string format
+            if (val != null && LABKEY.Utils.isDate(val)) {
+                val = new Date(val).format("Y-m-d");
```

Comment:
Do we not use user-configured date formats in the plots?

Reply:
Unfortunately not, I saw a few other places in this Luminex Levey-Jennings plot code which were just using this hardcoded date format string. I'll add some TODOs, but not address it as part of this PR.